ConfigSpace/hyperparameters.pyx

```diff
@@ -258,6 +288,44 @@ cdef class Constant(Hyperparameter):
                       transform: bool = False) -> List:
         return []
 
+    def pdf(self, vector: np.ndarray) -> np.ndarray:
+        """
+        Computes the probability density function of the parameter in 
+        the original parameter space (the one specified by the user).
+        For each hyperparameter type, there is also a method _pdf which 
+        operates on the transformed (and possibly normalized) parameter
+        space. Only legal values return a positive probability density,
+        otherwise zero.
+        
+        Parameters
+        ----------
+        vector: np.ndarray
+            the (N, ) vector of imputs for which the probability density
+            function is to be computed.
+        """
+        if vector.ndim != 1:
+            raise ValueError("Method pdf expects a one-dimensional numpy array")
+        return self._pdf(vector)
```

Comment:
@mfeurer 
Since inverse_transform returns a scalar in all cases, we can't. You'd have to redefine transform_scalar to return the same shape as the input (which seems reasonable to me), but I opted to do this instead. Happy to change it if you're OK with it.

Reply:
So now I've worked around it instead. It's not as consistent as the alternative you propose (and why we can't have a unified pdf function for all HPs), but it yields correct results.